packages/convenience/test/Convenience.sol.js

```diff
@@ -1,46 +1,259 @@
 const { expect } = require("chai");
+const { ethers } = require("hardhat");
 
+let mockApi3Voting, mockApi3Token, api3Pool, convenience;
 let roles;
-let convenience, mockApi3Pool, mockPrimaryVoting, mockSecondaryVoting;
-const VotingAppType = Object.freeze({ Primary: 0, Secondary: 1 });
+
+const VOTER_STATE = ["ABSENT", "YEA", "NAY"].reduce((state, key, index) => {
+  state[key] = index;
+  return state;
+}, {});
+
+const VOTING_TYPE = ["Primary", "Secondary"].reduce((state, key, index) => {
+  state[key] = index;
+  return state;
+}, {});
```

Comment:
The enum declaration above looks much nicer, I don't know why you went this. Also, even though these are defined as constants, semantically they are enums and enum names tend to be CamelCase.

Reply:
The [older test cases](https://github.com/api3dao/api3-dao/blob/a0b1db882126fb4233fb9dcd59d496c2d0a80eba/packages/api3-voting/test/voting.js#L14-L17) used this convention. So I kept the same convention.